src/client/datascience/dataScienceSurveyBanner.ts

```diff
@@ -59,7 +63,10 @@ export class DataScienceSurveyBannerLogger implements IInteractiveWindowListener
                     .updateValue(state.value + args.cellIds.length)
                     .then(() => {
                         // On every update try to show the banner.
-                        return this.dataScienceSurveyBanner.showBanner();
+                        return Promise.all([
+                            this.dataScienceSurveyBanner.showBanner(),
+                            this.insidersNativeNotebooksSurveyBanner.showBanner()
```

Comment:
Won't we end up with two prompts?

Reply:
No because the data science survey is only enabled in stable, and the other one is only enabled in insiders.